packages/terra-clinical-item-collection/src/_ListView.jsx

```diff
@@ -39,10 +39,13 @@ function createListItems(children, onSelect, requiredElements) {
 const ListView = ({
   children, onSelect, isListDivided, requiredElements,
 }) => {
+  const hasAChevron = React.Children.toArray(children).some(child => child.props.showListItemChevron);
+
   const listItems = createListItems(children, onSelect, requiredElements);
+  const roleSpread = hasAChevron ? { role: 'listbox' } : {};
```

Comment:
Why are we only applying this role when `hasAChevron` is true?  Should we not apply this at all times?

Reply:
If the list has `role="listbox"` but doesn't have a child with `role="option"` then that is considered an accessibility violation. `hasAChevron` will be true only if at least one child of the list has `role="option"`.

Reply:
I'm not super familiar with list accessibility rules, so should we be adding `role="option"` to each list item regardless of the chevron?

Reply:
According to the [specification for the option role](https://www.w3.org/TR/2010/WD-wai-aria-20100916/roles#option) it is for
> A selectable item in a select list.

so I suppose we should be checking for the `isSelectable` prop here and in `terra-list` instead.